app/database_etl/airtable_records_handler/airtable_records_formatter.py

```diff
@@ -79,8 +84,12 @@ def standardize_airtable_data(df: pd.DataFrame) -> pd.DataFrame:
         df[col] = df[col].apply(lambda x: x[0] if x is not None else x)
 
     # Convert elements that are "Not reported" or "Not Reported" or "NR" to None
-    df.replace({'nr': None, 'NR': None, 'Not Reported': None, 'Not reported': None,
-                'Not available': None, 'NA': None}, inplace=True)
+    df.replace({'nr': None, 'NR': None, 'Not Reported': None, 'Not reported': None, 'Not available': None,
+                'NA': None, 'N/A': None}, inplace=True)
+
+    # Antibody target reported are multi select and will have ['Not reported'], ['NR'] - convert to None
+    # Can also have a situation with ["Spike", "Not reported"] so need to check within list
+    df['antibody_target'] = df['antibody_target'].apply(lambda x: replace_null_fields(x))
```

Comment:
shouldn't line 87 do what line 92 does? I'm a bit confused

Reply:
no it doesn't because the antibody target column is like: ["Not reported", "Spike], ["Nucleocapsid], ["Spike, Nucleocapsid"] so we need to look inside the lists within the rows to replace the Not reported